openwisp_radius/api/urls.py

```diff
@@ -62,4 +62,9 @@ def get_api_urls(api_views=None):
             api_views.change_phone_number,
             name='phone_number_change',
         ),
+        url(
+            r'^(?P<slug>[\w-]+)/radiusbatch/(?P<radbatch>[\w-]+)/pdf/$',
```

Comment:
let's change this to `pk`

Reply:
I would start using path and use the uuid converter: https://docs.djangoproject.com/en/3.0/ref/urls/#path
See also here: openwisp/openwisp-firmware-upgrader#84

The converters are handy because they allow us to avoid having to implement checks ourselves (try supplying a bogus UUID with the current code and see what happens).

Reply:
To change this, I'll have to change `DispatchOrgMixin` used by other urls, i.e change other urls as well -- on it. :smile: